apps/web/src/components/Dropdown/components/Option.tsx

```diff
@@ -0,0 +1,28 @@
+import { ListboxOption } from "@headlessui/react";
+
+import type { Option as OptionProps } from "../Dropdown";
+
+export const Option: React.FC<OptionProps> = function ({ label, value }) {
+  return (
+    <ListboxOption
+      className={({ active }) =>
```

Comment:
`active` is deprecated.ts(6385)
listbox.d.ts(62, 9): The declaration was marked as deprecated here

```typescript
type OptionRenderPropArg = {
    /** @deprecated use `focus` instead */
    active: boolean;
    focus: boolean;
    selected: boolean;
    disabled: boolean;
    selectedOption: boolean;
};
```